Home page 02

ValentinCop opened the pull request.

ValentinCop committed:
working on home page

ValentinCop committed:
working on home-page (front)

ValentinCop committed:
styled home-page

ValentinCop committed:
Merge branch 'dev' into home-page-02

ValentinCop committed:
worked on the responsive of home page

Commit:
Update GlobalStyle.jsx

delete useless line

Commit:
Update index.jsx

transform CONNEXION to connexion

Commit:
Update index.jsx

Change name of Style by SHome and classname who by about

Commit:
Update style.jsx

change class who by about

LuigiMenez commented on frontend/src/pages/Home/style.jsx:
```diff
@@ -2,27 +2,117 @@ import styled from "styled-components";
-    font-size: 1.5rem;
-    font-family: var(--main-font);
-    text-align: center;
+  .about {
```
Possibility to make `.about, .connection p{display: none;}`

SDecambray replied:
Donec

LuigiMenez reviewed (changes requested):
Modified style

Commit:
Update style.jsx

reduce some useless line

.about, 
 .connection p, 
 .banner :not(.logo) {
    display: none;
  }

LuigiMenez reviewed (approved):
Ok for me.

LoicBrassart reviewed (approved):
SnE
